extensions/mssql/package.json

```diff
@@ -9,6 +9,9 @@
   "engines": {
     "vscode": "*"
   },
+  "enabledApiProposals": [
+    "textDocumentNotebook"
```

Comment:
Hm, we don't have the api proposal stuff they moved to yet. Is this part actually needed?

Reply:
Yes, it exposes the **notebook** property of TextDocuments

Reply:
Never mind, just needed to import the typings file